hack/ci/e2e-k8s.sh

```diff
@@ -86,6 +86,21 @@ create_cluster() {
     esac
   fi
 
+  scheduler_extra_args="      \"v\": \"${KIND_CLUSTER_LOG_LEVEL}\""
+  if [ -n "${SCHEDULER_LOG_FORMAT:-}" ]; then
```

Comment:
I think we should probably just phase out the KUBELET_LOG_FORMAT in favor of like CLUSTER_LOG_FORMAT and apply it to as many components as possible for the current version?

at some point we want to eliminate this script in favor of a real tool, expanding the "API" surface will make it less managable.

Reply:
do you mean replacing KUBELET_LOG_FORMAT and SCHEDULER_LOG_FORMAT with CLUSTER_LOG_FORMAT?

Reply:
/cc @BenTheElder

Reply:
yes, but possible also support KUBELET_LOG_FORMAT until we can migrate CI.

sorry we were on holiday here, also I'm going to have limited availability the next ~two weeks